server/lib/logging/route_logging.js

```diff
@@ -13,24 +13,39 @@ const config = require('../configuration');
  * Enhances connect logger middleware - custom formats.
  * See lib/configuration for usage.
  */
-const formats = {
-  'default_fxa': ':remote-addr - - ":method :url HTTP/:http-version" :status :response-time :res[content-length] ":referrer" ":user-agent"',
-
-  'dev_fxa': ':method :url :status :response-time'
```

Comment:
let's try to use the string one for `dev_fxa`, the new `default_fxa` looks great!